modules/metering-install-verify.adoc

```diff
@@ -61,4 +61,4 @@ pod-usage-cpu-cores                          2019-08-05T16:52:00Z   2019-08-05T1
 pod-usage-memory-bytes                       2019-08-05T16:52:00Z   2019-08-05T18:08:00Z   2019-08-05T16:52:00Z   2019-08-05T18:08:00Z   2019-08-05T18:54:20Z   9m49s
 ----
 
-Once all Pods are ready and you have verified that data is being imported, you can begin using Metering to collect data and report on your cluster.
+Once all Pods are ready and you have verified that data is being imported, you can begin using metering to collect data and report on your cluster.
```

Comment:
s/Once/After
> ! once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”